pipelines/datapackage/update-datapackage.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# [wf] execute update-datapackage.sh stage
```

Comment:
looks like you combined two features on the same PR, was that intentional? If it was, it's not a problem, I just want to double check